modules/Bio/EnsEMBL/IO/Object/GFF3.pm

```diff
@@ -0,0 +1,73 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+Bio::EnsEMBL::IO::Object::GFF3 - Generic object for holding GFF3 based records
+
+=head1 SYNOPSIS
+
+  use Bio::EnsEMBL::IO::Object::GFF3;
+  use Bio::EnsEMBL::IO::Parser::GFF3;
+
+  my $parser = Bio::EnsEMBL::IO::Parser::GFF3->open('myfile.gff3');
+  my $obj = Bio::EnsEMBL::IO::Object::GFF3->new($parser->get_fields);
+
+  $obj->munrge_seqname('my_seq');
+
+=head1 Description
+
+An object derived from ColumnBasedGeneric with a specialized create_record to properly
+format the record as GFF3 and map the strand to the correct values type (+/-)
+rather than GFF3 as the generic object does.
+
+=cut
+
+package Bio::EnsEMBL::IO::Object::GFF3;
+
+use base qw/Bio::EnsEMBL::IO::Object::ColumnBasedGeneric/;
```

Comment:
We are supposed to use 'use parent' now, aren't we? Plus several other files included in this PR use 'use parent' so it would be nice to be consistent.